fighters/koopajr/src/opff.rs

```diff
@@ -39,26 +39,22 @@ unsafe fn kart_jump_waveland(boma: &mut BattleObjectModuleAccessor, status_kind:
     }
 }
 
-unsafe fn upB_kart_respawn(boma: &mut BattleObjectModuleAccessor, status_kind: i32) {
-    if WorkModule::is_flag(boma, *FIGHTER_KOOPAJR_INSTANCE_WORK_ID_FLAG_SPECIAL_HI_INTERRUPT)
-    && boma.is_situation(*SITUATION_KIND_GROUND) {
-        boma.change_status_req(*FIGHTER_KOOPAJR_STATUS_KIND_SPECIAL_HI_LANDING, false);
-    }
-
-    if boma.is_status(*FIGHTER_KOOPAJR_STATUS_KIND_SPECIAL_HI_LANDING) && boma.is_prev_status(*FIGHTER_STATUS_KIND_ITEM_THROW) {
-        KineticModule::clear_speed_all(boma);
-    }
```

Comment:
this seems separate from the up b kart respawning - clears speed on landing from kart after throwing an item?

Reply:
This was used to fix an old bug with AGT out of upB, no longer needed anymore